### PR TITLE
Fix for not showing disabled plugins

### PR DESCRIPTION
**Pull request by w20k**
Related to issue: #3905

### CODE REVIEW

File: modules/backend/classes/BackendController.php [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
-use File;
-use View;
+use Closure;
 use Config;
-use Response;
+use File;
 use Illuminate\Routing\Controller as ControllerBase;
 use October\Rain\Router\Helper as RouterHelper;
-use Closure;
+use Response;
```
Comment (LukeTowers) [inline modules/backend/classes/BackendController.php]: Why did you change the order of imports? @w20k

Comment (w20k) [reply modules/backend/classes/BackendController.php]: No particular reason. Already thought about that, after commit.
But for me it's just default Intellij Idea formatting, when set to auto :)

Comment (LukeTowers) [reply modules/backend/classes/BackendController.php]: Can you revert it? October's import statements aren't organized that way so I see no reason to change them willy nilly :)

Comment (w20k) [reply modules/backend/classes/BackendController.php]: Sure thing :) Sorry, had a glass of wine before pull, so let it slip somehow :D

Comment (LukeTowers) [reply modules/backend/classes/BackendController.php]: No worries, take care of it when able :) I appreciate all your help on the project so far!

Comment (w20k) [reply modules/backend/classes/BackendController.php]: If you could be still available in few min will make PR update ;)
